Examples/Search/Search/SearchApp.swift

```diff
@@ -3,14 +3,20 @@ import SwiftUI
 
 @main
 struct SearchApp: App {
+  
+  @MainActor
+  static let store = Store(initialState: Search.State()) {
+    Search()
+      ._printChanges()
+  }
+  
   var body: some Scene {
     WindowGroup {
-      SearchView(
-        store: Store(initialState: Search.State()) {
-          Search()
-            ._printChanges()
-        }
-      )
+      if isTesting {
+        EmptyView()
+      } else {
+        SearchView(store: Self.store)
+      }
```

Comment:
Since we're not doing UI tests I think we can simplify:

```suggestion
      SearchView(store: Self.store)
```